### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ScreenReader: VoiceOver does not read the state of the "Toggle Replace" button as expanded/collapsed.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52314]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : High Sierra 10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader  : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Activity Bar and select "Search" button.
3. Navigate to "Toggle Replace" button. Use "Enter/Space Bar" to expand/collapse.

**Actual:**
VoiceOver does not read the state of the "Toggle Replace" button as expanded/collapsed.

**Expected:**
VoiceOver should read the state of the "Toggle Replace" button as expanded/collapsed.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will not know about the state of the button and its use(display/hide content).

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewSearch_VoiceOver_ToggleButtonState.pptx](https://github.com/Microsoft/vscode/files/2115295/A11y_VSCode_ViewSearch_VoiceOver_ToggleButtonState.pptx)

Does this issue occur when all extensions are disabled?: Yes
